### PR TITLE
Adds in event for Add Calendar Links

### PR DESCRIPTION
**Pull request by roman-bytes**

### CODE REVIEW

File: src/schedule/2021-02-08-add-calendar-links-to-11ty-with-computed-data.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+---
+title: Add Calendar Links to 11ty with Computed Data
+type: Mob Job
+date: 2021-02-08T01:00:00-05:00
```
Comment (BenDMyers) [inline src/schedule/2021-02-08-add-calendar-links-to-11ty-with-computed-data.md:4]: Thank you for submitting an event, @roman-bytes!

This is currently showing up as 10 **pm** PST - I'm guessing you want 10 **am** PST?

```suggestion
date: 2021-02-08T13:00:00-05:00
```

Come to think of it, we should probably make it a little clearer that the CLI is expecting a 24-hour format 😅

Comment (roman-bytes) [reply src/schedule/2021-02-08-add-calendar-links-to-11ty-with-computed-data.md]: whoops, lets not do this at 1am. Should be fixed now.

**Comment by BenDMyers** [comment]
@all-contributors please add @roman-bytes for content

**Review by BenDMyers — APPROVED** [review]
This looks great! Please feel free to merge this in whenever you're ready!

**Pull request merged**